### PR TITLE
SAK-41411 Highlight student grade within gradebook charts

### PR DESCRIPTION
**Pull request by amiedavis**

### CODE REVIEW

File: edu-services/gradebook-service/impl/src/java/org/sakaiproject/component/gradebook/GradebookFrameworkServiceImpl.java [hunk 1]
```diff
@@ -72,7 +72,9 @@ public void addGradebook(final String uid, final String name) {
             // templates, so if for some reason none have been defined yet,
             // do that now.
             if (gradingScales.isEmpty()) {
-                if (log.isInfoEnabled()) log.info("No Grading Scale defined yet. This is probably because you have upgraded or you are working with a new database. Default grading scales will be created. Any customized system-wide grade mappings you may have defined in previous versions will have to be reconfigured.");
+                if (log.isInfoEnabled()) {
+					log.info("No Grading Scale defined yet. This is probably because you have upgraded or you are working with a new database. Default grading scales will be created. Any customized system-wide grade mappings you may have defined in previous versions will have to be reconfigured.");
+				}
```
Comment (ottenhoff) [inline edu-services/gradebook-service/impl/src/java/org/sakaiproject/component/gradebook/GradebookFrameworkServiceImpl.java]: tabs/spacing looks incorrect

File: edu-services/gradebook-service/impl/src/java/org/sakaiproject/component/gradebook/GradebookFrameworkServiceImpl.java [hunk 2]
```diff
@@ -298,15 +315,20 @@ private void mergeGradeMappings(Collection gradingScaleDefinitions, Session sess
...
 	}
 
 
+	@Override
 	public void deleteGradebook(final String uid)
 		throws GradebookNotFoundException {
-        if (log.isDebugEnabled()) log.debug("Deleting gradebook uid=" + uid + " by userUid=" + getUserUid());
+        if (log.isDebugEnabled()) {
+			log.debug("Deleting gradebook uid=" + uid + " by userUid=" + getUserUid());
+		}
```
Comment (ottenhoff) [inline edu-services/gradebook-service/impl/src/java/org/sakaiproject/component/gradebook/GradebookFrameworkServiceImpl.java]: tabs/spacing

**Comment by steveswinsburg** [comment]
GradebookFrameworkServiceImpl wasn't actually touched in this change but appears to have been since master has moved on and the diff has wigged out. @amiedavis once you merge in master and resolve these conflicts this should resolve itself. I made some changes to these files in the previous PR for this feature 👍

**Comment by amiedavis** [comment]
@steveswinsburg this is good to go now, sorry about the mess I didn't realise I had gotten so far behind master and then had some issues fixing my merge conflicts.

**Pull request merged**